pkg/testutil/fixture.go

```diff
@@ -86,11 +86,13 @@ func ObjectStoreProfileOrSkip(c *check.C, osType objectstore.ProviderType, locat
 	var key, val string
 	switch osType {
 	case objectstore.ProviderTypeS3:
-		if region, ok := os.LookupEnv(awsconfig.Region); ok && location.Region == "" {
-			location.Region = region
-		}
-		if endpoint, ok := os.LookupEnv("LOCATION_ENDPOINT"); ok && location.Endpoint == "" {
-			location.Endpoint = endpoint
+		if useMinio, ok := os.LookupEnv("USE_MINIO"); ok && useMinio == "true" {
+			if region, ok := os.LookupEnv(awsconfig.Region); ok && location.Region == "" {
+				location.Region = region
+			}
+			if endpoint, ok := os.LookupEnv("LOCATION_ENDPOINT"); ok && location.Endpoint == "" {
+				location.Endpoint = endpoint
+			}
 		}
```

Comment:
We also want to set it for S3, right?

Reply:
@varuni-dev what if we use different S3-compatible store? We won't be able to override the values to use that. If we want to use different Region and Endpoint for the store